core/src/main/cpp/android.cpp

```diff
@@ -185,6 +185,11 @@ bool Android::DisableProfileSaver() {
                                               : version < kS ? "_ZN3art12ProfileSaver20ProcessProfilingInfoEbPt"
                                               : "_ZN3art12ProfileSaver20ProcessProfilingInfoEbbPt";
             process_profiling_info = handle.GetSymbolAddress(symbol);
+
+            // Android 15 QPR1
+            if (!process_profiling_info && version >= kV) {
```

Comment:
I prefer to not check android version here, as ART is currently a mainline APEX module and updated ART module can be back-ported to Android S+ devices through a Google Play Update
```suggestion
            if (!process_profiling_info) {
```